esmvaltool/recipes/ref/recipe_model_benchmarking_timeseries_region.yml

```diff
@@ -0,0 +1,342 @@
+# ESMValTool
+---
+documentation:
+  title: Benchmarking of a single model for a region through extract_shape.
+  description: >
+    Benchmarking: time series.
+  authors:
+    - lauer_axel
+    - bock_lisa
+    - hassler_birgit
+    - lindenlaub_lukas
+    - schlund_manuel
+    - weigel_katja
+  maintainer:
+    - lauer_axel
+  references:
+    - lauer25gmd
+  projects:
+    - dlrmabak
+
+
+datasets:
+  # Dataset to be benchmarked
+  - {dataset: MIROC6, grid: gn, benchmark_dataset: true}
+
+preprocessors:
+  pp_abs:
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_shape:
+      shapefile: ar6
+      crop: true
+      decomposed: true
+      ids:
+        Name:
+          - N.Europe
+    area_statistics:
+      operator: mean
+    multi_model_statistics:
+      span: overlap
+      statistics:
+        - operator: percentile
+          percent: 10
+        - operator: percentile
+          percent: 90
+      exclude: [reference_dataset, MIROC6]
+
```

Comment:
The lines above we probably don't need if we do not show the CMIP6 model envelope, no?